### PR TITLE
Update German localization (part 1)

### PR DESCRIPTION
**Pull request by surfdude29**
Supersedes #3664

Following the same process as #2917 (thanks [DeepL](https://www.deepl.com/translator)! 🙏) I have updated some of the German translations in this PR. I also made a few tweaks to existing strings, but of course let me know if I got something wrong there.

There are currently 715 missing strings in the German message catalog, but I thought  that was too many to tackle properly in one PR. So I've done the first 200+ and I'll move on to the rest in later PRs.

I signed up for a trial of DeepL Pro which allows you to specify an informal tone, so the translated strings in this PR should at least follow that.

I incorporated the corrections from @PythooonUser made in commit https://github.com/bluesky-social/social-app/pull/3664/commits/659bc5e78d0582551afb97cfc5d94a84063fc91a but otherwise I thought it was easiest to start from scratch with this PR.

All corrections from native German speakers are welcome, especially previous translators including @PythooonUser, @cdfzo and @imbstt.

Thanks! :)

EDIT: I've also merged in some updates to existing translations, thanks again @cdfzo! 🙏

### CODE REVIEW

File: src/locale/locales/de/messages.po [hunk 1]
```diff
@@ -558,28 +459,20 @@ msgstr "Eine E-Mail wurde an deine vorherige Adresse {0} gesendet. Sie enthält
 
 #: src/components/dialogs/GifSelect.tsx:252
 msgid "An error occured"
-msgstr ""
+msgstr "Es ist ein Fehler aufgetreten"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:462]:
```suggestion
msgstr "Ein Fehler ist aufgetreten"
```

both work but chatgpt says `Empfehlung: Für eine klare und direkte Kommunikation, insbesondere in technischen oder informellen Kontexten, klingt "Ein Fehler ist aufgetreten" besser.` and I agree

File: src/locale/locales/de/messages.po [hunk 2]
```diff
@@ -598,7 +491,7 @@ msgstr "Es ist ein Problem aufgetreten, bitte versuche es erneut."
 
 #: src/screens/Onboarding/StepInterests/index.tsx:218
 msgid "an unknown error occurred"
-msgstr ""
+msgstr "Es ist ein unbekannter Fehler aufgetreten"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:494]:
```suggestion
msgstr "Ein unbekannter Fehler ist aufgetreten"
```

File: src/locale/locales/de/messages.po [hunk 3]
```diff
@@ -647,29 +540,16 @@ msgstr "App-Passwörter"
...
 
 #: src/components/moderation/LabelsOnMeDialog.tsx:227
 #: src/screens/Messages/Conversation/ChatDisabled.tsx:91
 msgid "Appeal submitted"
-msgstr ""
-
-#: src/components/moderation/LabelsOnMeDialog.tsx:193
-#~ msgid "Appeal submitted."
-#~ msgstr "Anfechtung abgeschickt."
+msgstr "Anfechtung abgeschickt"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:552]:
```suggestion
msgstr "Anfechtung gesendet"
```

again, both work, but this sounds better imo

File: src/locale/locales/de/messages.po [hunk 4]
```diff
@@ -678,50 +558,38 @@ msgstr ""
...
 msgstr "Erscheinungsbild"
 
 #: src/screens/Feeds/NoSavedFeedsOfAnyType.tsx:47
 #: src/screens/Home/NoFeedsPinned.tsx:93
 msgid "Apply default recommended feeds"
-msgstr ""
+msgstr "Standardmäßig empfohlene Feeds anwenden"
 
 #: src/screens/StarterPack/StarterPackScreen.tsx:610
 msgid "Are you sure you want delete this starter pack?"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:571]: there's a `to` missing in the english translation lol

File: src/locale/locales/de/messages.po [hunk 5]
```diff
@@ -678,50 +558,38 @@ msgstr ""
...
-
-#: src/components/dms/ConvoMenu.tsx:189
-#~ msgid "Are you sure you want to leave this conversation? Your messages will be deleted for you, but not for other participants."
-#~ msgstr ""
+msgstr "Möchtest du diese Nachricht wirklich löschen? Die Nachricht wird für dich gelöscht, nicht jedoch für den anderen Teilnehmer."
 
 #: src/components/dms/LeaveConvoPrompt.tsx:48
 msgid "Are you sure you want to leave this conversation? Your messages will be deleted for you, but not for the other participant."
-msgstr ""
+msgstr "Möchtest du dieses Gespräch wirklich verlassen? Deine Nachrichten werden für dich gelöscht, nicht jedoch für den anderen Teilnehmer."
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:584]:
```suggestion
msgstr "Möchtest du diese Konversation wirklich verlassen? Deine Nachrichten werden für dich gelöscht, nicht jedoch für den anderen Teilnehmer."
```

File: src/locale/locales/de/messages.po [hunk 6]
```diff
@@ -678,50 +558,38 @@ msgstr ""
...
 
 #: src/view/com/feeds/FeedSourceCard.tsx:314
 msgid "Are you sure you want to remove {0} from your feeds?"
 msgstr "Bist du sicher, dass du {0} von deinen Feeds entfernen möchtest?"
 
 #: src/components/FeedCard.tsx:306
 msgid "Are you sure you want to remove this from your feeds?"
-msgstr ""
+msgstr "Bist du sicher, dass du diesen von deinen Feeds entfernen möchtest?"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:592]:
```suggestion
msgstr "Bist du sicher, dass du dies von deinen Feeds entfernen möchtest?"
```

File: src/locale/locales/de/messages.po [hunk 7]
```diff
@@ -922,30 +758,26 @@ msgstr "Bücher"
 
 #: src/components/FeedInterstitials.tsx:281
 msgid "Browse more accounts on the Explore page"
-msgstr ""
+msgstr "Stöbere auf der Seite „Explore” nach weiteren Konten"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:761]: the page is probably not called `Explore` in german

File: src/locale/locales/de/messages.po [hunk 8]
```diff
@@ -922,30 +758,26 @@ msgstr "Bücher"
...
 
 #: src/screens/Home/NoFeedsPinned.tsx:103
 #: src/screens/Home/NoFeedsPinned.tsx:109
 msgid "Browse other feeds"
-msgstr ""
-
-#: src/view/screens/Settings/index.tsx:893
-#~ msgid "Build version {0} {1}"
-#~ msgstr "Build-Version {0} {1}"
+msgstr "Stöbere in anderen Feeds"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:780]:
```suggestion
msgstr "Andere Feeds durchsuchen"
```

File: src/locale/locales/de/messages.po [hunk 9]
```diff
@@ -1103,81 +923,61 @@ msgstr "Deine E-Mail ändern"
...
+msgstr "Chat-Einstellungen"
 
 #: src/components/dms/ConvoMenu.tsx:84
 msgid "Chat unmuted"
-msgstr ""
-
-#: src/screens/Messages/Conversation/index.tsx:26
-#~ msgid "Chat with {chatId}"
-#~ msgstr ""
+msgstr "Chat wurde nicht mehr stummgeschaltet"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:947]:
```suggestion
msgstr "Chatstummschaltung aufgehoben"
```

File: src/locale/locales/de/messages.po [hunk 10]
```diff
@@ -1514,25 +1271,17 @@ msgstr "Fortfahren mit {0} (aktuell angemeldet)"
 
 #: src/view/com/post-thread/PostThreadLoadMore.tsx:52
 msgid "Continue thread..."
-msgstr ""
+msgstr "Fortsetzung des Threads…"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1274]:
```suggestion
msgstr "Thread fortsetzen…"
```

*Fortsetzung* is *continuation*

File: src/locale/locales/de/messages.po [hunk 11]
```diff
@@ -1620,7 +1365,7 @@ msgstr "Urheberrechtsbestimmungen"
 
 #: src/components/dms/LeaveConvoPrompt.tsx:39
 msgid "Could not leave chat"
-msgstr ""
+msgstr "Konntest den Chat nicht verlassen"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1368]:
```suggestion
msgstr "Du konntest den Chat nicht verlassen"
```

File: src/locale/locales/de/messages.po [hunk 12]
```diff
@@ -1630,21 +1375,13 @@ msgstr "Feed konnte nicht geladen werden"
...
-
-#: src/components/dms/ConvoMenu.tsx:68
-#~ msgid "Could not unmute chat"
-#~ msgstr ""
+msgstr "Chat konnte nicht stummgeschaltet werden"
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:272
 msgid "Create"
-msgstr ""
+msgstr "Erstelle"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1384]:
```suggestion
msgstr "Erstellen"
```

File: src/locale/locales/de/messages.po [hunk 13]
```diff
@@ -1676,15 +1413,15 @@ msgstr "Konto erstellen"
...
 
 #: src/screens/Onboarding/StepProfile/index.tsx:283
 msgid "Create an avatar instead"
-msgstr ""
+msgstr "Erstelle stattdessen einen Avatar"
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:172
 msgid "Create another"
-msgstr ""
+msgstr "Erstelle ein weitere"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1424]:
```suggestion
msgstr "Ein weiteres erstellen"
```

File: src/locale/locales/de/messages.po [hunk 14]
```diff
@@ -1571,20 +1320,20 @@ msgstr "Kopieren"
 
 #: src/view/com/modals/ChangeHandle.tsx:474
 msgid "Copy {0}"
-msgstr "{} kopieren"
+msgstr "Kopiere {0}"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1323]:
```suggestion
msgstr "{} kopieren"
```

no change required

File: src/locale/locales/de/messages.po [hunk 15]
```diff
@@ -1676,15 +1413,15 @@ msgstr "Konto erstellen"
 #: src/components/dialogs/Signin.tsx:86
 #: src/components/dialogs/Signin.tsx:88
 msgid "Create an account"
-msgstr ""
+msgstr "Erstelle ein Konto"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1416]:
```suggestion
msgstr "Ein Konto erstellen"
```

File: src/locale/locales/de/messages.po [hunk 16]
```diff
@@ -1657,17 +1394,17 @@ msgstr "Erstelle ein neues Bluesky-Konto"
...
 #: src/components/StarterPack/ProfileStarterPacks.tsx:259
 #: src/Navigation.tsx:338
 msgid "Create a starter pack"
-msgstr ""
+msgstr "Erstelle ein Starterpaket"
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:246
 msgid "Create a starter pack for me"
-msgstr ""
+msgstr "Erstelle ein Starterpaket für mich"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1407]:
```suggestion
msgstr "Ein Starterpaket für mich erstellen"
```

File: src/locale/locales/de/messages.po [hunk 17]
```diff
@@ -1657,17 +1394,17 @@ msgstr "Erstelle ein neues Bluesky-Konto"
...
 msgid "Create a QR code for a starter pack"
-msgstr ""
+msgstr "Erstelle einen QR-Code für ein Starterpaket"
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:165
 #: src/components/StarterPack/ProfileStarterPacks.tsx:259
 #: src/Navigation.tsx:338
 msgid "Create a starter pack"
-msgstr ""
+msgstr "Erstelle ein Starterpaket"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1403]:
```suggestion
msgstr "Ein Startpaket erstellen"
```

File: src/locale/locales/de/messages.po [hunk 18]
```diff
@@ -1657,17 +1394,17 @@ msgstr "Erstelle ein neues Bluesky-Konto"
 
 #: src/components/StarterPack/QrCodeDialog.tsx:151
 msgid "Create a QR code for a starter pack"
-msgstr ""
+msgstr "Erstelle einen QR-Code für ein Starterpaket"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1397]:
```suggestion
msgstr "QR-Code für ein Starterpaket erstellen"
```

File: src/locale/locales/de/messages.po [hunk 19]
```diff
@@ -1676,15 +1413,15 @@ msgstr "Konto erstellen"
 #: src/components/dialogs/Signin.tsx:86
 #: src/components/dialogs/Signin.tsx:88
 msgid "Create an account"
-msgstr ""
+msgstr "Erstelle ein Konto"
 
 #: src/screens/Onboarding/StepProfile/index.tsx:283
 msgid "Create an avatar instead"
-msgstr ""
+msgstr "Erstelle stattdessen einen Avatar"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1420]:
```suggestion
msgstr "Stattdessen einen Avatar erstellen"
```

File: src/locale/locales/de/messages.po [hunk 20]
```diff
@@ -1757,20 +1478,20 @@ msgstr "Dunkles Thema"
...
 #: src/screens/Settings/components/DeactivateAccountDialog.tsx:73
 #: src/view/screens/Settings/index.tsx:807
 msgid "Deactivate account"
-msgstr ""
+msgstr "Konto deaktivieren"
 
 #: src/view/screens/Settings/index.tsx:819
 msgid "Deactivate my account"
-msgstr ""
+msgstr "Deaktiviere mein Konto"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1490]:
```suggestion
msgstr "Mein Konto deaktivieren"
```

`Deaktiviere mein Konto` means `YOU must deactivate my account`, `Mein Konto deaktivieren` means `I deactivate my account`

File: src/locale/locales/de/messages.po [hunk 21]
```diff
@@ -1757,20 +1478,20 @@ msgstr "Dunkles Thema"
...
 
 #: src/view/screens/Settings/index.tsx:874
 msgid "Debug Moderation"
-msgstr ""
+msgstr "Fehlerbehebung Moderation"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1494]:
```suggestion
msgstr "Debug-Moderation"
```

last time, we translated `Debug panel` as `Debug-Panel`, so I'd also use *debug* here

File: src/locale/locales/de/messages.po [hunk 22]
```diff
@@ -1790,13 +1511,9 @@ msgstr "Löschen"
...
 msgstr "Konto löschen"
 
-#: src/view/com/modals/DeleteAccount.tsx:87
-#~ msgid "Delete Account"
-#~ msgstr "Konto löschen"
-
 #: src/view/com/modals/DeleteAccount.tsx:105
 msgid "Delete Account <0>\"</0><1>{0}</1><2>\"</2>"
-msgstr ""
+msgstr "Lösche Konto <0>„</0><1>{0}</1><2>”</2>"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1516]:
```suggestion
msgstr "Konto <0>„</0><1>{0}</1><2>”</2> löschen"
```

File: src/locale/locales/de/messages.po [hunk 23]
```diff
@@ -1809,23 +1526,23 @@ msgstr "App-Passwort löschen?"
 #: src/view/screens/Settings/index.tsx:891
 #: src/view/screens/Settings/index.tsx:894
 msgid "Delete chat declaration record"
-msgstr ""
+msgstr "Lösche den Datensatz für die Chat-Erklärung"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1529]:
```suggestion
msgstr "Datensatz für die Chat-Erklärung löschen"
```

File: src/locale/locales/de/messages.po [hunk 24]
```diff
@@ -1809,23 +1526,23 @@ msgstr "App-Passwort löschen?"
...
 
 #: src/components/dms/MessageMenu.tsx:124
 msgid "Delete for me"
-msgstr ""
+msgstr "Lösche für mich"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1533]:
```suggestion
msgstr "Für mich löschen"
```

File: src/locale/locales/de/messages.po [hunk 25]
```diff
@@ -1809,23 +1526,23 @@ msgstr "App-Passwort löschen?"
...
 
 #: src/components/dms/MessageMenu.tsx:147
 msgid "Delete message"
-msgstr ""
+msgstr "Nachricht löschen"
 
 #: src/components/dms/MessageMenu.tsx:122
 msgid "Delete message for me"
-msgstr ""
+msgstr "Lösche die Nachricht für mich"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1545]:
```suggestion
msgstr "Nachricht für mich löschen"
```

File: src/locale/locales/de/messages.po [hunk 26]
```diff
@@ -1890,27 +1607,19 @@ msgstr "Dimmen"
 
 #: src/components/dms/MessagesNUX.tsx:88
 msgid "Direct messages are here!"
-msgstr ""
+msgstr "Direkte Nachrichten sind angekommen!"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1610]:
```suggestion
msgstr "Direktnachrichten sind da!"
```

File: src/locale/locales/de/messages.po [hunk 27]
```diff
@@ -1890,27 +1607,19 @@ msgstr "Dimmen"
...
 
 #: src/view/screens/AccessibilitySettings.tsx:107
 msgid "Disable autoplay for GIFs"
-msgstr ""
+msgstr "Deaktiviere die automatische Wiedergabe für GIFs"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1614]:
```suggestion
msgstr "Automatische Wiedergabe für GIFs deaktivieren"
```

File: src/locale/locales/de/messages.po [hunk 28]
```diff
@@ -1890,27 +1607,19 @@ msgstr "Dimmen"
...
 
 #: src/view/screens/Settings/DisableEmail2FADialog.tsx:90
 msgid "Disable Email 2FA"
-msgstr ""
+msgstr "Deaktiviere die Zwei-Faktor-Authentifizierung per E-Mail"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1618]:
```suggestion
msgstr "Zwei-Faktor-Authentifizierung per E-Mail deaktivieren"
```

File: src/locale/locales/de/messages.po [hunk 29]
```diff
@@ -1890,27 +1607,19 @@ msgstr "Dimmen"
...
-msgstr ""
-
-#: src/view/screens/Settings/index.tsx:697
-#~ msgid "Disable haptics"
-#~ msgstr ""
-
-#: src/view/screens/Settings/index.tsx:697
-#~ msgid "Disable vibrations"
-#~ msgstr ""
+msgstr "Deaktiviere haptische Rückmeldung"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1622]:
```suggestion
msgstr "Haptische Rückmeldung deaktivieren"
```

File: src/locale/locales/de/messages.po [hunk 30]
```diff
@@ -1940,7 +1645,7 @@ msgstr "Apps daran hindern, abgemeldeten Nutzern mein Konto zu zeigen"
 
 #: src/tours/HomeTour.tsx:70
 msgid "Discover learns which posts you like as you browse."
-msgstr ""
+msgstr "„Discover” lernt beim Durchstöbern, welche Beiträge dir gefallen."
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1648]:
```suggestion
msgstr "„Discover” lernt beim Browsen, welche Beiträge dir gefallen."
```

I don't like *Durchstöbern*

File: src/locale/locales/de/messages.po [hunk 31]
```diff
@@ -1949,19 +1654,19 @@ msgstr "Entdecke neue benutzerdefinierte Feeds"
...
 
 #: src/components/ProgressGuide/List.tsx:40
 msgid "Dismiss getting started guide"
-msgstr ""
+msgstr "Anleitung zum Einstieg schließen"
 
 #: src/view/screens/AccessibilitySettings.tsx:95
 msgid "Display larger alt text badges"
-msgstr ""
+msgstr "Zeige größeren Alt-Text-Badges"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1669]:
```suggestion
msgstr "Größere Alt-Text-Badges zeigen"
```

File: src/locale/locales/de/messages.po [hunk 32]
```diff
@@ -2149,22 +1846,17 @@ msgstr "Profil bearbeiten"
...
+msgstr "Starterpaket bearbeiten"
 
 #: src/view/com/modals/CreateOrEditList.tsx:234
 msgid "Edit User List"
 msgstr "Benutzerliste bearbeiten"
 
 #: src/components/WhoCanReply.tsx:127
 msgid "Edit who can reply"
-msgstr ""
+msgstr "Bearbeite, wer antworten kann"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1859]:
```suggestion
msgstr "Bearbeiten, wer antworten kann"
```

File: src/locale/locales/de/messages.po [hunk 33]
```diff
@@ -2176,7 +1868,7 @@ msgstr "Bearbeite deine Profilbeschreibung"
 
 #: src/Navigation.tsx:343
 msgid "Edit your starter pack"
-msgstr ""
+msgstr "Bearbeite dein Starterpaket"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1871]:
```suggestion
msgstr "Dein Starterpaket bearbeiten"
```

File: src/locale/locales/de/messages.po [hunk 34]
```diff
@@ -2185,7 +1877,7 @@ msgstr "Bildung"
 
 #: src/components/dialogs/ThreadgateEditor.tsx:98
 msgid "Either choose \"Everybody\" or \"Nobody\""
-msgstr ""
+msgstr "Entweder „Alle” oder „Niemand” auswählen"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1880]:
```suggestion
msgstr "Wähle entweder „Alle” oder „Niemand” aus"
```

File: src/locale/locales/de/messages.po [hunk 35]
```diff
@@ -2194,7 +1886,7 @@ msgstr "E-Mail"
 
 #: src/view/screens/Settings/DisableEmail2FADialog.tsx:64
 msgid "Email 2FA disabled"
-msgstr ""
+msgstr "Zwei-Faktor-Authentifizierung per E-Mail wurde deaktiviert"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1889]:
```suggestion
msgstr "2FA per E-Mail wurde deaktiviert"
```

File: src/locale/locales/de/messages.po [hunk 36]
```diff
@@ -2219,17 +1911,17 @@ msgstr "E-Mail:"
 
 #: src/components/dialogs/Embed.tsx:112
 msgid "Embed HTML code"
-msgstr ""
+msgstr "Bette HTML-Code ein"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1914]:
```suggestion
msgstr "HTML-Code einbetten"
```

File: src/locale/locales/de/messages.po [hunk 37]
```diff
@@ -2219,17 +1911,17 @@ msgstr "E-Mail:"
...
 #: src/view/com/util/forms/PostDropdownBtn.tsx:324
 #: src/view/com/util/forms/PostDropdownBtn.tsx:326
 msgid "Embed post"
-msgstr ""
+msgstr "Beitrag einbetten"
 
 #: src/components/dialogs/Embed.tsx:101
 msgid "Embed this post in your website. Simply copy the following snippet and paste it into the HTML code of your website."
-msgstr ""
+msgstr "Einbetten diesen Beitrag in deine Website ein. Kopiere einfach den folgenden Code und füge ihn in den HTML-Code deiner Website ein."
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:1924]:
```suggestion
msgstr "Bette diesen Beitrag in deine Website ein. Kopiere einfach den folgenden Code und füge ihn in den HTML-Code deiner Website ein."
```

File: src/locale/locales/de/messages.po [hunk 38]
```diff
@@ -2454,85 +2129,72 @@ msgstr "Das App-Passwort konnte nicht erstellt werden."
...
-#: src/screens/Messages/Conversation/MessageListError.tsx:29
-#~ msgid "Failed to send message(s)."
-#~ msgstr ""
+msgstr "Konnte nicht gesendet werden"
 
 #: src/components/moderation/LabelsOnMeDialog.tsx:223
 #: src/screens/Messages/Conversation/ChatDisabled.tsx:87
 msgid "Failed to submit appeal, please try again."
-msgstr ""
+msgstr "Anfechten nicht eingereicht. Bitte erneut versuchen."
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:2184]:
```suggestion
msgstr "Anfechtung nicht eingereicht. Bitte versuche es erneut."
```

File: src/locale/locales/de/messages.po [hunk 39]
```diff
@@ -2543,13 +2205,9 @@ msgstr "Feed"
...
 msgstr "Feed von {0}"
 
-#: src/view/screens/Feeds.tsx:709
-#~ msgid "Feed offline"
-#~ msgstr "Feed offline"
-
 #: src/components/StarterPack/Wizard/WizardListCard.tsx:55
 msgid "Feed toggle"
-msgstr ""
+msgstr "Kippschalter zum Feed"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:2210]:
```suggestion
msgstr "Feed-Umschalter"
```

maybe

File: src/locale/locales/de/messages.po [hunk 40]
```diff
@@ -2608,23 +2258,11 @@ msgstr "Konten zum Folgen finden"
 
 #: src/tours/HomeTour.tsx:88
 msgid "Find more feeds and accounts to follow in the Explore page."
-msgstr ""
+msgstr "Weitere Feeds und Konten, denen du folgen kannst, findest du auf der Seite „Explore”."
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:2261]:
```suggestion
msgstr "Finde weitere Feeds und Konten, denen du folgen kannst, auf der „Explore” Seite."
```

File: src/locale/locales/de/messages.po [hunk 41]
```diff
@@ -2636,11 +2274,11 @@ msgstr "Passe die Diskussionsstränge an."
 
 #: src/screens/StarterPack/Wizard/index.tsx:192
 msgid "Finish"
-msgstr ""
+msgstr "Beende"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:2277]:
```suggestion
msgstr "Beenden"
```

File: src/locale/locales/de/messages.po [hunk 42]
```diff
@@ -2636,11 +2274,11 @@ msgstr "Passe die Diskussionsstränge an."
...
 
 #: src/tours/Tooltip.tsx:149
 msgid "Finish tour and begin using the application"
-msgstr ""
+msgstr "Beende die Tour und beginne mit der Nutzung der Anwendung"
```
Comment (cdfzo) [inline src/locale/locales/de/messages.po:2281]:
```suggestion
msgstr "Tour beenden und mit der Nutzung der Anwendung beginnen"
```

**Review by cdfzo — COMMENTED** [review]
we should decide for either `Startpaket` or `Starterpaket`

**Comment by surfdude29** [comment]
> we should decide for either `Startpaket` or `Starterpaket`

Thanks so much for all your corrections @cdfzo!

Before I go through them I think this should be decided, which do you think sounds better?

File: src/locale/locales/de/messages.po (hunk 7 above)
Comment (surfdude29) [reply src/locale/locales/de/messages.po:761]: I'll have to double-check this, my understanding is that it's not been translated (like `Following`)

File: src/locale/locales/de/messages.po (hunk 14 above)
Comment (surfdude29) [reply src/locale/locales/de/messages.po:1323]: This was one of the corrections from @PythooonUser, but you think it's better the other way around?

File: src/locale/locales/de/messages.po [hunk 43]
```diff
@@ -8,267 +8,210 @@ msgstr ""
...
 #: src/screens/Profile/Header/Metrics.tsx:23
 msgid "{0, plural, one {follower} other {followers}}"
-msgstr ""
+msgstr "{0, plural, one {Follower} other {Follower}}"
 
 #: src/components/ProfileHoverCard/index.web.tsx:402
 #: src/screens/Profile/Header/Metrics.tsx:27
 msgid "{0, plural, one {following} other {following}}"
-msgstr ""
+msgstr "{0, plural, one {Folge ich} other {Folge ich}}"
```
[the remark this answers is not in the document]
Comment (cdfzo) [reply src/locale/locales/de/messages.po:48]: ok, good point

File: src/locale/locales/de/messages.po [hunk 44]
```diff
@@ -8,267 +8,210 @@ msgstr ""
...
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:266
 msgid "{0, plural, one {Like (# like)} other {Like (# likes)}}"
```
[the remark this answers is not in the document]
Comment (cdfzo) [reply src/locale/locales/de/messages.po:51]: ahh, makes sense, thanks!

File: src/locale/locales/de/messages.po [hunk 45]
```diff
@@ -8,267 +8,210 @@ msgstr ""
...
 
-#: src/screens/StarterPack/Wizard/index.tsx:485
-#~ msgid "<0>{0} </0>and<1> </1><2>{1} </2>are included in your starter pack"
-#~ msgstr ""
-
 #: src/screens/StarterPack/Wizard/index.tsx:466
 msgctxt "profiles"
 msgid "<0>{0}, </0><1>{1}, </1>and {2, plural, one {# other} other {# others}} are included in your starter pack"
-msgstr ""
+msgstr "<0>{0}, </0><1>{1} </1>und {2, plural, one {# weitere Person} other {# weitere Personen}} sind in deinem Starterpaket enthalten"
```
[the remark this answers is not in the document]
Comment (cdfzo) [reply src/locale/locales/de/messages.po:173]: I'd remove it

File: src/locale/locales/de/messages.po (hunk 14 above)
Comment (cdfzo) [reply src/locale/locales/de/messages.po:1323]: here's the context of `src/view/com/modals/ChangeHandle.tsx:474`:
```ts
Copy {isDNSForm ? _(msg`Domain Value`) : _(msg`File Contents`)}
```

I'd say `Dateiinhalt kopieren` sounds better than `Kopiere Dateiinhalt`

Comment (surfdude29) [reply src/locale/locales/de/messages.po:1323]: I'll just fix the missing variable then
```suggestion
msgstr "{0} kopieren"
```

**Comment by surfdude29** [comment]
> Let me know which you think sounds better and then I'll change the strings accordingly :)

@cdfzo Is `Startpaket` better maybe?

**Review by pfrazee — APPROVED** [review]
Thanks all!

**Pull request merged**